src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,93 @@
+package duke.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Parent class of all Task types. ToDo, Deadline, Event.
+ */
+public abstract class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Constructor for Task object.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+    public String getStatusIcon() {
```

Comment:
Missing JavaDocs for the public methods getStatusIcon, markAsDone and unmarkAsDone